MANIFEST.in

```diff
@@ -1,8 +1,23 @@
-#include *.txt
+include AUTHORS.md
+include LICENSE
+include README.md
+
 include versioneer.py
 include pvlib/_version.py
```

Comment:
AFAIK, this line is not necessary, since `pvlib` is a package, and is already listed in `setup.py`